app/scripts/migrations/120.1.ts

```diff
@@ -0,0 +1,37 @@
+import { cloneDeep } from 'lodash';
+import { hasProperty } from '@metamask/utils';
+import { InternalAccount } from '@metamask/keyring-api';
+
+type VersionedData = {
+  meta: { version: number };
+  data: Record<string, unknown>;
+};
+
+export const version = 120.1;
+
+/**
+ * Add a default value for importTime in the InternalAccount
+ *
+ * @param originalVersionedData
+ */
+export async function migrate(
+  originalVersionedData: VersionedData,
+): Promise<VersionedData> {
+  const versionedData = cloneDeep(originalVersionedData);
+  versionedData.meta.version = version;
+  transformState(versionedData.data);
+  return versionedData;
+}
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function transformState(state: Record<string, any>) {
+  if (!hasProperty(state, 'UserStorageController')) {
```

Comment:
Pre-existing users would be expected to have no `UserStorageController` state, since this is the first time this controller is introduced. Is that true? We should be expecting this condition to be `true` in all cases. This is the case where we want to set `isProfileSyncingEnabled` to `null`.

Reply:
This is interesting. The PR that it was introduced was 11.16 (but this label may be incorrect).
https://github.com/MetaMask/metamask-extension/pull/23353

I think you are right though, we can add a check for if it does exist too. I'll update the tests as well.

Reply:
Oh interesting, so it was introduced before it was used, and not visible in settings during that time. OK, if that's true, then you're right. We'll need to handle both cases.